platform/iphone/os_iphone.mm

```diff
@@ -220,6 +220,34 @@
 	perform_event(ev);
 };
 
+void OSIPhone::pencil_press(int p_idx, int p_x, int p_y, bool p_pressed, bool p_doubleclick) {
+	Ref<InputEventMouseButton> ev;
+	ev.instance();
+	ev->set_button_index(1);
+	ev->set_pressed(p_pressed);
+	ev->set_position(Vector2(p_x, p_y));
+	ev->set_global_position(Vector2(p_x, p_y));
+	ev->set_doubleclick(p_doubleclick);
+	//ev->set_pressure(p_force);
+	//ev->set_tilt(p_tilt);
```

Comment:
I'd remove the commented out code. If it's meant as a reminder to implement pressure and tilt support, it should be a `// TODO` or `// FIXME` comment explaining what needs to be done.

Reply:
Ok, done!